sources/deny.toml

```diff
@@ -13,7 +13,7 @@ allow = [
     # "CC0-1.0",
     "ISC",
     "MIT",
-    "OpenSSL",
+    # "OpenSSL",
```

Comment:
If this isn't needed, is it possible to remove the license?

Reply:
For licenses that we are comfortable to approve unconditionally, we leave them as comments here.